generate.sh

```diff
@@ -62,16 +62,6 @@ then
 fi
 ln -fs "$yq_filename" yq
 
-# tap-fitter doesn't have binary downloads at the moment, so assume golang is
```

Comment:
tap-fitter leverages the composite template, but earlier in the thread there was discussion of the devfile going away, so I'm not sure how much impact that actually represents.

Reply:
This is ok by me. I had to remove the call to tap-fitter later on before the script was able to run, since the program is now missing